test/e2e/specs/specs-playwright/wp-media__upload-spec.ts

```diff
@@ -39,36 +39,36 @@ describe( DataHelper.createSuiteTitle( 'Media: Upload' ), () => {
 	`( 'Upload media files ($siteType)', ( { user } ) => {
 		let mediaPage: MediaPage;
 
-		it( 'Log In', async function () {
-			const loginFlow = new LoginFlow( page, user );
-			await loginFlow.logIn();
+		it( 'Log in', async function () {
+			const loginPage = new LoginPage( page );
+			await loginPage.login( { account: user } );
 		} );
 
-		it( 'Navigate to Media', async function () {
-			const sidebarComponent = new SidebarComponent( page );
-			await sidebarComponent.navigate( 'Media' );
-		} );
+		// it( 'Navigate to Media', async function () {
+		// 	const sidebarComponent = new SidebarComponent( page );
+		// 	await sidebarComponent.navigate( 'Media' );
+		// } );
 
-		it( 'See media gallery', async function () {
-			mediaPage = new MediaPage( page );
-		} );
+		// it( 'See media gallery', async function () {
+		// 	mediaPage = new MediaPage( page );
+		// } );
 
-		it( 'Upload image and confirm addition to gallery', async () => {
-			await mediaPage.upload( testFiles.image.fullpath );
-		} );
+		// it( 'Upload image and confirm addition to gallery', async () => {
+		// 	await mediaPage.upload( testFiles.image.fullpath );
+		// } );
 
-		it( 'Upload audio and confirm addition to gallery', async () => {
-			await mediaPage.upload( testFiles.audio.fullpath );
-		} );
+		// it( 'Upload audio and confirm addition to gallery', async () => {
+		// 	await mediaPage.upload( testFiles.audio.fullpath );
+		// } );
 
-		it( 'Upload an unsupported file type and see the rejection notice', async function () {
-			try {
-				await mediaPage.upload( testFiles.unsupported.fullpath );
-			} catch ( error: unknown ) {
-				if ( error instanceof Error ) {
-					assert.match( error.message, /could not be uploaded/i );
-				}
-			}
-		} );
+		// it( 'Upload an unsupported file type and see the rejection notice', async function () {
```

Comment:
Are these examples that are currently failing due to the refactor, and will be eventually worked out before merging? Or what was the reason to comment those out?

Reply:
This was for stress testing of the changes.

Reply:
Removed in 94ca9bd42a5816208ce94c9248a11e7ec69a6d57.